src/components/dialogs/dataSources/COVIDcast.svelte

```diff
@@ -7,10 +7,13 @@
 
   export let id: string;
 
+  let api_key = '';
   let data_source = '';
   let signal = '';
   let geo_type = '';
   let geo_value = '';
+  let form_key = '';
+  let invalid_key = false;
```

Comment:
This will help readability by preventing double negatives: 
```suggestion
  let valid_key = true;
```